lib/meterReadingUtils.js

```diff
@@ -0,0 +1,17 @@
+import { getMeterReadingsandInvoicesByIds } from '../airtable/request';
+
+export const getLatestMeterReadingForCustomer = async (customer) => {
+    const meterReadingIds = customer.meterReadingIds;
+
+    if (!meterReadingIds || meterReadingIds.length === 0) {
+        return null
+    }
+
+    // Get Latest Meter Reading for customer
+    try {
+        const latestMeterReadings = await getMeterReadingsandInvoicesByIds(meterReadingIds, '', [{ field: "Date", direction: "desc" }]);
+        return latestMeterReadings.length > 0 ? latestMeterReadings[0] : null;
+    } catch (error) {
+        console.error('Error occurred while fetching the latest meter reading for customer: ', error);
```

Comment:
Ties into a comment on your frontend PR but I think it'd generally be helpful to have more details in this error msg about where this occurs e.g. `(getLatestMeterReadingForCustomer) Error ...`

Reply:
Good point, thanks!